qml/pages/AppPage.qml

```diff
@@ -115,9 +115,22 @@ Page {
                 visible: app.changelog
                 //% "Changelog"
                 text: qsTrId("orn-changelog")
-                onClicked: pageStack.push(Qt.resolvedUrl("ChangelogPage.qml"), {
-                                              changelog: app.changelog
-                                          })
+                onClicked: {
+                    // try to extract the url of the first link from the changelog
+                    var regex = /^(.*)\<a href\=\"(http[s]?:\/\/[^\s]*)\"[\>]*\>[^\<]*\<\/a\>(.*)$/g;
+                    var parts = regex.exec(app.changelog);
+                    if (parts !== null && parts.length === 4) {
+                        // if url found and the non-URL part of the changelog is short
+                        // for e.g. "see: <a href="https://github.com/">...
+                        // then open the URL automatically
+                        if ((parts[1].length + parts[3].length) < 32)
```

Comment:
Can you please unite these two conditional expressions and remove semicolons to make the code consistent with the other parts?

Reply:
Fixed!